### ISSUE TITLE
Feature Request: Invert the logic: Add "Enable area"

### ISSUE THREAD

**Issue by erffrfez** [bradcfisher/OctoPrint-ExcludeRegionPlugin#76]
I have used this add-on quite often, and THANKS.
There have been times when I've wanted to exclude all but (eg) 3 parts on a bed (the ones that failed with the initial print). Perhaps one part broke, or didn't stick so need to print that again, but also need to deal with the 2 other parts that then got a whole bunch of added 'spaghetti'. This can be quite a lengthy process to do in the plugin, adding multiple exclude areas.
I'd like to be able to mark a big area in red (exclude it as can do now), then mark just the parts to be printed in green (re-enable them for printing.)
Instead of doing this:
![Screenshot from 2023-04-24 15-00-53](https://user-images.githubusercontent.com/7607672/233925313-21d34080-b669-41aa-a2ad-ad8e2b6c810d.png)

we'd do four circles, one to turn things off, then individually turn them on again:
![Screenshot from 2023-04-24 15-21-02b](https://user-images.githubusercontent.com/7607672/234174552-c27427d7-db0a-4c32-bb52-6432afc99edb.png)

For consideration please.  :)

**Comment by semicolonmystery** [comment]
I totally agree it would be great if something like this would be implemented.
